### DISCUSSION TITLE
Newbie Struggling

### DISCUSSION THREAD

**Discussion by mdpotter15** [AAndyProgram/SCrawler#83] [Q&A] [answered]
Hello! First off, I want to thank you for making this tool. Unfortunately, I am struggling to get it to work for me. I have gone through the whole wiki and am still puzzled.

My objective is to download all the pictures and videos from a single instagram account: grippingfoodwithforce.

Here's what it looks like when I add the account:

![image](https://user-images.githubusercontent.com/117503906/200106001-d3317aa7-604d-4d20-875e-07ef687c9489.png)

After adding, the icon is blank and hitting "Download" doesn't seem to work. It just says the download is complete without actually doing it.

Please help me! What am I doing wrong? Is there an idiot-proof set of steps you can give me for my specific scenario?

**Comment by AAndyProgram** [comment, marked as answer]
1. Your LOG button is red. I guess it's a good color to indicate that something is wrong.
2. https://github.com/AAndyProgram/SCrawler/wiki/Settings#instagram-requirements
